app/controllers/manuals_controller.rb

```diff
@@ -5,33 +5,49 @@ class ManualsController < ApplicationController
 
   before_action :ensure_manual_is_found
   before_action :ensure_document_is_found, only: :show
-  before_action :load_manual
   before_action :prevent_robots_from_indexing_hmrc_manuals
 
   def index
-    set_expiry(manual)
+    set_expiry(content_store_manual)
+
+    render :index, locals: {
+      presented_manual: presented_manual
+    }
   end
 
   def show
-    set_expiry(document)
-    @document = Document.new(document, @manual)
+    set_expiry(document_from_repository)
+    document = Document.new(document_from_repository, manual)
+
+    render :show, locals: {
+      presented_manual: presented_manual,
```

Comment:
I understand your point and originally in an uncommitted version I did leave it as manual. My reasoning to change it to a `presented_manual` rather than `manual` is to make it easier to understand that there might be other methods we can use in the views (from `ManualPresenter`) rather than just what is defined in `Manual`.

For example, when we call `presented_manual.breadcrumbs`, it's clearer where that method definition is coming from.

But if you think it would be better to change it back to a `manual` I can do that.